doc/snippets/Microsoft.Data.SqlClient/SqlConfigurableRetryFactory.xml

```diff
@@ -0,0 +1,98 @@
+|10928|Resource ID: %d. The %s limit for the database is %d and has been reached. For more information, see http://go.microsoft.com/fwlink/?LinkId=267637.|
+|10060|An error has occurred while establishing a connection to the server. When connecting to SQL Server, this failure may be caused by the fact that under the default settings SQL Server does not allow remote connections. (provider: TCP Provider, error: 0 - A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.) (Microsoft SQL Server, Error: 10060)|
+|10054|The data value for one or more columns overflowed the type used by the provider.|
+|10053|Could not convert the data value due to reasons other than sign mismatch or overflow.|
+|997|A connection was successfully established with the server, but then an error occurred during the login process. (provider: Named Pipes Provider, error: 0 - Overlapped I/O operation is in progress)|
+|233|A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.) (Microsoft SQL Server, Error: 233)|
+|64||
+|20||
+|0||
+
+]]>
+            </format>
+            </remarks>
+        </SqlConfigurableRetryFactory>
+        <CreateExponentialRetryProvider>
+            <param name="retryLogicOption">An object of <see cref="T:Microsoft.Data.SqlClient.FloatingRetryLogicOption" /> carries the configurations for the object creation.</param>
+            <summary>Provides an exponential time interval retry logic provider.</summary>
+            <returns>A <see cref="T:Microsoft.Data.SqlClient.SqlRetryLogicBaseProvider" /> object.</returns>
+            <remarks>
+				<format type="text/markdown"><![CDATA[
+
+> [NOTE!]
+> The inner enumerator includes the randomization to prevent multiple instances of the client sending subsequent retry attempts at the same time.
+
+]]></format>
+			</remarks>
+            <exception cref="T:System.ArgumentNullException">If the `retryLogicOption` parameter was null.</exception>
+        </CreateExponentialRetryProvider>
+        <CreateIncrementalRetryProvider>
+            <param name="retryLogicOption">An object of <see cref="T:Microsoft.Data.SqlClient.FloatingRetryLogicOption" /> carries the configurations for the object creation.</param>
+            <summary>Provides an incremental time interval retry logic provider.</summary>
+            <returns>A <see cref="T:Microsoft.Data.SqlClient.SqlRetryLogicBaseProvider" /> object.</returns>
+            <remarks>
+				<format type="text/markdown"><![CDATA[
+
+> [NOTE!]
+> The inner enumerator includes the randomization to prevent multiple instances of the client sending subsequent retry attempts at the same time.
```

Comment:
```suggestion
> The inner enumerator includes randomization to prevent multiple instances of the client from performing subsequent retry attempts at the same time.
```